uclapi.openapi.json

```diff
@@ -0,0 +1,4256 @@
+            "description": "Whether bookings in this room will be confirmed automatically. A stands for automated, and N for not automated. P represents that the confirmation will be automatic, but only under certain circumstances."
+          },
+          "location" : {
+            "type" : "object",
+            "properties" : {
+              "address" : {
+                "type" : "array",
+                "minItems" : 4,
+                "maxItems" : 4,
+                "items" : {
+                  "type" : "string"
+                },
+                "example" : [ "Gower Street", "London", "WC1E 6BT" ],
+                "description": "Address represented as an array."
+              },
+              "coordinates" : {
+                "type" : "object",
+                "properties" : {
+                  "lat" : {
+                    "type" : "number",
+                    "example" : "51.524699",
+                    "description": "Latitude of location"
+                  },
+                  "lng" : {
+                    "type" : "number",
+                    "example" : "-0.13366",
+                    "description": "Longitude of location"
+                  }
+                }
+              }
+            }
+          }
+        }
+      },
+      "booking" : {
+        "properties" : {
+          "slotid" : {
+            "type" : "number",
+            "example" : 998503,
+            "description": "Slot ID."
+          },
+          "end_time" : {
+            "type" : "string",
+            "example" : "2016-09-02T18:00:00+00:00",
+            "description": "End time of the event."
+          },
+          "description" : {
+            "type" : "string",
+            "example" : "split weeks to assist rooming 29.06",
+            "description": "Describes what the booking is. Could contain a module code (for example WIBRG005) or just the type of activity (for example Lecture)."
+          },
+          "roomname" : {
+            "type" : "string",
+            "example" : "Torrington (1-19) 433",
+            "description": "The name of the room. It often includes the name of the site (building) as well."
+          },
+          "siteid" : {
+            "type" : "number",
+            "example" : 86.0,
+            "description": "Every room is inside a site (building). All sites have IDs."
+          },
+          "contact" : {
+            "type" : "string",
+            "example" : "Ms Leah Markwick",
+            "description": "The name of the person who made the booking. Substrings of the contact name can be used: Queries for Mark will include Mark Herbster. Sometimes, a society or student group may be the contact for a booking."
+          },
+          "weeknumber" : {
+            "type" : "number",
+            "example" : 1,
+            "description": "Timetabled week number."
+          },
+          "roomid" : {
+            "type" : "number",
+            "example" : "433",
```

Comment:
nit: number in quotes

Reply:
this is also actually a string, updated!